charts/spinnaker/templates/configmap/halyard-config.yaml

```diff
@@ -90,10 +90,12 @@ data:
     {{ end }}
 
     # Docker Registry
-    $HAL_COMMAND config provider docker-registry enable
+    {{ $skipValidation := 0 }}
+    {{ range $index, $registry := .Values.dockerRegistries }}{{ if $registry.noValidate }}{{ $skipValidation = 1 }}{{ end }}{{ end }}
```

Comment:
Can be shorter, but stayed consistent in code style:

```suggestion
    {{ range .Values.dockerRegistries }}{{ if .noValidate }}{{ $skipValidation = 1 }}{{ end }}{{ end }}
```

Reply:
Putting it in the helpers template SGTM. Would rename the variable then. Couldn't come up with a good name, something like `shouldSkipGlobalDockerRegistryValidation`? 🤔

Reply:
Nice suggestion. TIL about Helm's introduced `include` directive. Verified that this works by rebasing this on top of the `pdsoels/fix-deployment` branch and running `helmfile diff`. Also validated the flag is not set when `noValidate` is not set or set to false. Will push your suggestion with some other things.